src/vs/workbench/services/configuration/browser/configurationService.ts

```diff
@@ -462,7 +462,7 @@ export class WorkspaceService extends Disposable implements IWorkbenchConfigurat
 
 			if (!this.localUserConfiguration.hasTasksLoaded) {
 				// Reload local user configuration again to load user tasks
-				runWhenIdle(() => this.reloadLocalUserConfiguration(), 5000);
+				this._register(runWhenIdle(() => this.reloadLocalUserConfiguration(), 5000));
```

Comment:
I am not sure since I did not write this code, but is this adding disposables over and over?

Reply:
It apepared to be called only once from `initialize`, but leaving it off to @sandy081 to decide.

Reply:
Makes sense